src/libostree/libostree-released.sym

```diff
@@ -14,9 +14,7 @@
   Lesser General Public License for more details.
  
   You should have received a copy of the GNU Lesser General Public
-  License along with this library; if not, write to the
```

Comment:
This changes the overall digest of the file, thus I think you'll have to update `tests/test-symbols.sh` too, see
https://github.com/ostreedev/ostree/blob/a07b8d6c0076d424555c37b6fa462ce279b831c0/tests/test-symbols.sh#L58-L63

Reply:
Thank you @lucab, fixed in latest push.